eland/etl.py

```diff
@@ -490,6 +498,28 @@ def csv_to_eland(  # type: ignore
     if chunksize is None:
         kwargs["chunksize"] = DEFAULT_CHUNK_SIZE
 
+    # Bug in Pandas v1.3.0
+    # If names and prefix both passed as None, it's considering them as specified values and throwing ValueError
+    # Ref: https://github.com/pandas-dev/pandas/issues/42387
+    if PANDAS_VERSION >= "1.3.0":
+        if kwargs["names"] is None and kwargs["prefix"] is None:
+            kwargs.pop("prefix")
+        if kwargs["warn_bad_lines"] is True:
```

Comment:
Let's separate the `if kwargs["names"] / "prefix" ...` section and the `if kwargs["warn_bad_lines"] / "error_bad_lines"...` section so we can tell they're different.